creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -666,6 +673,8 @@ async function secondarySyncFromPrimary({
   )
   const metricEndTimerFn = secondarySyncFromPrimaryMetric.startTimer()
 
+  const mode = forceResyncConfig?.forceResync ? 'force_resync' : 'default'
```

Comment:
can we be more specific than `default` for the not force resyncing flow? like `force_resync_enabled` and `force_resync_disabled`

Reply:
i initially was gonna do that then decided to leave it as an array so we can add more label values in future without having to change the existing ones from boolean format - wdyt

Reply:
yeah this is fine